hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -14,96 +14,109 @@ Please see the [boot EEPROM](booteeprom.md) page for more information about the
-The retry counters are reset when switching to the next boot mode.
+The minimum supported bootloader version for custom boot modes is 2020-09-03.
 
-`BOOT_ORDER` fields  
+##### `BOOT_ORDER` fields
 The BOOT_ORDER property defines the sequence for the different boot modes. It is read right to left and up to 8 digits may be defined.
 
-| Value | Mode         | Description                                                                                       |
-|-------|--------------|---------------------------------------------------------------------------------------------------|
-|  0x1  |  SD CARD     |  SD card (or eMMC on Compute Module 4)                                                            |
-|  0x2  |  NETWORK     |  Network boot                                                                                     |
-|  0x3  |  RPIBOOT     |  RPIBOOT -  See [usbboot](https://github.com/raspberrypi/usbboot)  (since 2020-09-03)             |
-|  0x4  |  USB-MSD     |  USB mass storage boot (since 2020-09-03)                                                         |
-|  0x5  |  BCM-USB-MSD |  USB 2.0 boot from USB Type-C socket or USB Type-A socket on CM4 IO board.     (since 2020-12-14) |
-|  0x6  |  NVME        |  Boot from an NVMe SSD connected to the PCIe socket (CM4 only). See [NVMe boot](bootmodes/nvme.md) for more details
-|  0xe  |  STOP        |  Stop and display error pattern (since 2020-09-03). A power cycle is required to exit this state. |
-|  0xf  |  RESTART     |  Start again with the first boot order field. (since 2020-09-03)                                  |
-
-After trying each non-zero boot mode the bootloader stops. However, from 2020-09-03 the bootloader will monitor the SD card detect pin and try SD boot if a new SD card is inserted.
-
-Default: `0xf41`  
-
-* Boot mode `0x0` will retry the SD boot if the SD card detect pin indicates that the card has been inserted or replaced.
-* The default boot order is `0xf41` which means continuously try SD then USB mass storage.
-* `RPIBOOT` is intended for use with Compute Module 4 to load a custom debug image (e.g. a Linux RAM-disk) instead of the normal boot. This should be the last boot option because it does not currently support timeouts or retries.
-* `BCM-USB-MSD` boot allows a Compute Module 4 to boot from USB without requiring an additional PCIe XHCI card. This USB controller does not support USB 3.0 and booting will be slower than `USB-MSD` boot on a Pi 4 or Pi 400.
-
+| Value | Mode            | Description                                                                                                              |
+|-------|-----------------|--------------------------------------------------------------------------------------------------------------------------|
+|  0x0  |  SD CARD DETECT |  Try SD then wait for card-detect to indicate that the card has changed - deprecated now that 0xf (RESTART is available) |
```

Comment:
I suspect "0xf (RESTART is available)" was supposed to be "0xf (RESTART) is available" ?

Reply:
done